db/seeds/gobierto_seeds/gobierto_data/recipe.rb

```diff
@@ -0,0 +1,122 @@
+          end
+          frequency.name_translations = { ca: "Freqüència", en: "Frequency", es: "Frecuencia" }
+          frequency.position = 2
+          frequency.options = {
+            configuration: { vocabulary_type: "single_select" },
+            vocabulary_id: vocabulary.id.to_s
+          }
+          frequency.save
+        end
+
+        category = site.custom_fields.vocabulary_options.where(class_name: "GobiertoData::Dataset").find_or_initialize_by(uid: "category")
+        if category.new_record?
+          vocabulary = site.vocabularies.find_or_initialize_by(slug: "datasets-category")
+          if vocabulary.new_record?
+            vocabulary.name_translations = { ca: "Categoria de conjunt de dades", en: "Dataset Category", es: "Categoría de conjunto de datos" }
+            vocabulary.save
+            DEFAULT_CATEGORY_NAMES.each_with_index do |category_name, index|
+              vocabulary.terms.create(
+                name_translations: Hash[site.configuration.available_locales.product([category_name])],
+                position: index + 1
+              )
+            end
+          end
+          category.name_translations = { ca: "Categoria", en: "Category", es: "Categoría" }
+          category.position = 3
+          category.options = {
+            configuration: { vocabulary_type: "multiple_select" },
+            vocabulary_id: vocabulary.id.to_s
+          }
+          category.save
+        end
+
+        dataset_source_license = site.custom_fields.string.where(class_name: "GobiertoData::Dataset").find_or_initialize_by(uid: "dataset-source")
+        if dataset_source_license.new_record?
+          dataset_source_license.name_translations = { ca: "Origen", en: "Source", es: "Fuente" }
+          dataset_source_license.position = 4
+          dataset_source_license.options = { configuration: {} }
+          dataset_source_license.save
+        end
+
+        dataset_source_url = site.custom_fields.string.where(class_name: "GobiertoData::Dataset").find_or_initialize_by(uid: "dataset-source-url")
+        if dataset_source_url.new_record?
+          dataset_source_url.name_translations = { ca: "Enllaç a llicència", en: "Link to License", es: "Enlace a Licencia" }
+          dataset_source_url.position = 5
+          dataset_source_url.options = { configuration: {} }
+          dataset_source_url.save
+        end
+
+        licenses = site.custom_fields.vocabulary_options.where(class_name: "GobiertoData::Dataset").find_or_initialize_by(uid: 'dataset-license')
+        if licenses.new_record?
+          vocabulary = site.vocabularies.find_or_initialize_by(slug: "dataset-license")
+          if vocabulary.new_record?
+            vocabulary.name_translations = { ca: "Llicència", en: "License", es: "Licencia" }
+            vocabulary.save
+            vocabulary.terms.create(name_translations: { ca: "Open Data (ODbl)",   en: "Open Data (ODbl)",   es: "Open Data (ODbl)" }, position: 1)
```

Comment:
I'd add an option for not known (maybe the first option or default)

Reply:
can I name as:

ca: "Llicència desconeguda"
en: "Unknown license"
es: "Licencia desconocida"

Reply:
ok!

Reply:
I think it would be better to allow to not select a value, and if so, don't show the field.